### DISCUSSION TITLE
CommandBar

### DISCUSSION THREAD

**Discussion by workgroupengineering** [community/community#64209] [Mobile]
### Select Topic Area

Product Feedback

### Body

Added option to always show CommandBar for quick navigation.

**Comment by 9777410929** [comment]
#hi

**Reply by 9777410929** [reply]
hi

**Comment by Jaber-Saed** [comment]
It sounds like you're providing feedback about a feature that you've added or would like to see added to a product, possibly related to a CommandBar for quick navigation. This can be a helpful addition to improve user experience and accessibility. However, since you haven't mentioned the specific product you're referring to, I'll provide a general response. If you're talking about a particular software or platform, make sure to adapt the feedback accordingly:

Adding an option to always show a CommandBar for quick navigation is a valuable enhancement. Quick and accessible navigation is crucial for user productivity and ease of use. With an always-visible CommandBar, users can easily access commands and features without needing to locate or trigger a hidden navigation menu.

This feature can streamline user interactions, improve discoverability of functions, and enhance the overall user experience. Users who rely on keyboard shortcuts or those who prefer quick navigation will likely appreciate having the CommandBar readily available.

When implementing this feature, consider providing users with the ability to toggle the visibility of the CommandBar based on their preferences. Customization options empower users to tailor the interface to their workflow.

Feedback like this helps product teams understand user needs and preferences, ultimately leading to more user-friendly and efficient software. Remember to check if the product already has a feedback channel or community where you can submit your suggestion formally. Your input can contribute to shaping the future of the product.

**Reply by workgroupengineering** [reply]
sorry for the hermetic answer, I thought the Mobile channel was referring to the app only.

**Comment by Tolgahan0** [comment]
> Added option to always show CommandBar for quick navigation.

I'd be happy to help you fix the problem with the added option to always show the CommandBar for quick navigation. However, to provide you with the most accurate solution, I'll need more context about the problem you're experiencing. Could you please provide additional details about the issue you're facing? This could include:

1. **Description of the Problem:** Please describe the problem you're encountering in detail. What happens when you try to use the added option always to show the CommandBar? Is there an error message? Unexpected behavior?

2. **Steps to Reproduce:** Provide a step-by-step guide on how to reproduce the problem. This will help in understanding the issue better and finding a suitable solution.

3. **Technology Stack:** What programming languages, frameworks, and tools are you using for this feature? This information can help diagnose the problem.

4. **Code Snippets:** If applicable, provide relevant code snippets related to the added option and the CommandBar functionality. This will assist in identifying any coding issues.

5. **Expected Outcome:** What should happen when the option always shows the CommandBar is enabled? What is the expected behavior?

6. **Error Messages:** If there are any error messages or console logs related to this issue, please provide them.

7. **Browser/Environment Details:** If this issue is related to a web application, provide details about the browser you're using and any relevant environment information.

With this additional information, I can offer you a more accurate solution or suggestions to troubleshoot and resolve the problem you're facing.

**Comment by workgroupengineering** [comment]
# Use case **Mobile App** **v.1.126. (739)** os **Android 13**:

1. Go notifications list
2. Clink on issue
3. Click a link inside issue theth refer ad other github issue/pr/repo
4. explore linked resource o go to issue/pr/repo
5. To return to notications list there is two possily
    - terminate app and re run
    - press n times back until you go back to the notifications list  

If the command bar were visible, just click on the notification button to return to the list.
